storage/innobase/handler/handler0alter.cc

```diff
@@ -1069,10 +1069,17 @@ innobase_col_to_mysql(
 		field->reset();
 
 		if (field->type() == MYSQL_TYPE_VARCHAR) {
+			if (field->column_format() == COLUMN_FORMAT_TYPE_COMPRESSED) {
+				/* Skip compressed varchar column when
```

Comment:
This code path is never hit in my tests.
I believe this code is here because `innobase_col_to_mysql()` is adapted from `row_sel_field_store_in_mysql_format()` which has very similar structure.